packages/react-storage/src/components/StorageBrowser/composables/DataTable/dataCells/CheckboxDataCell.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import { Checkbox } from '../../../components/Checkbox';
+import { Checkbox } from '@aws-amplify/ui-react';
```

Comment:
@calebpollman ^ I did this b/c it was the quickest/easiest way. I can try to put the Checkbox in the components API if you want

Reply:
In theory, we could also do the same thing here as proposed above for ActionDestination and expose a prop on data table like we did the the loading indicator?

Reply:
There is a loading indicator prop on the data table??

Reply:
You know what, ignore me on that. I don't know what became of that PR, apparently.

Reply:
No. This is also in reference to a PR of mine which is still in-flight